.gitignore

```diff
@@ -10,7 +10,8 @@ node_modules
 /prisma/dev.sqlite-journal
 database.sqlite
 
-.env
+*.env
```

Comment:
Just double checking for my own benefit - this will also catch `.env`, right?

Reply:
No you are right we need to be explicit. :bad-chat-gpt: